packages/pf3-component-mapper/src/form-fields/date-time-picker/date-time-picker.js

```diff
@@ -7,6 +7,12 @@ import { createDisabledDays } from './helpers';
 
 import 'react-day-picker/lib/style.css';
 
+const getPlaceholder = (variant, placeholder) => placeholder
+  ? placeholder
+  : variant === 'date'
+    ? 'Click in this input to select date'
+    : 'Click in this input to select date and time';
```

Comment:
```suggestion
    ? 'Click in this input to select a date'
    : 'Click in this input to select a date and a time';
```

I'm not sure about the second `a` in the `a date and a time` part, maybe we need someone who actually speaks English :laughing:

Reply:
They're both okay from a grammatical standpoint, both the original and the suggestion.